include/seqan3/alphabet/structure/structured_aa.hpp

```diff
@@ -26,8 +26,9 @@ namespace seqan3
 
 /*!\brief A seqan3::alphabet_tuple_base that joins an aminoacid alphabet with a protein structure alphabet.
  * \ingroup structure
- * \implements seqan3::Alphabet
- * \implements seqan3::detail::ConstexprAlphabet
+ * \implements seqan3::WritableAlphabet
+ * \implements seqan3::WritableAlphabet
+ * \if DEV \implements seqan3::detail::WritableConstexprAlphabet \endif
  * \implements seqan3::TriviallyCopyable
  * \implements seqan3::StandardLayout
  * \tparam sequence_alphabet_t Type of the first aminoacid letter; must satisfy seqan3::Alphabet.
```

Comment:
```suggestion
  * \tparam sequence_alphabet_t Type of the first aminoacid letter; must satisfy seqan3::WritableAlphabet and must be an object type.
```
This is a suggestion to get rid of the negation. But not sure if it improves as normal readers might not know the definition of object types. Maybe this must be then linked to cppreference. But as I said just a suggestion, no hard feelings if you drop this. but add then at least that they must not be reference types.

Reply:
>  But not sure if it improves as normal readers might not know the definition of object types.

Yeah, I think they wouldn't know. Also !ref is not the same as object (although we don't allow the other non-object types either). 

> no hard feelings if you drop this. but add then at least that they must not be reference types.

I have added a reference to the requirement of alphabet_tuple_base so that one doesn't have to copy'n'paste all the stuff.